test/it/client-listUsers.js

```diff
@@ -97,16 +112,16 @@ describe('client.listUsers().each()', () => {
         setTimeout(reject.bind(null, 'foo error'), 1000);
       });
     }).catch((err)=>{
-      assert.equal(localCount, 1);
-      assert.equal(err, 'foo error');
+      expect(localCount).to.equal(1);
+      expect(err).to.equal('foo error');
     });
   });
 });
 
 describe('client.listUsers().next()', () => {
   it('should return User models', () => {
     return client.listUsers().next().then(result => {
-      assert.instanceOf(result.value, models.User);
+      expect(result.value).to.be.an.instanceof(models.User);
```

Comment:
Yes, I have that on my radar. This was Robert's original test where we didn't use async await feature. Will convert this in a later PR.